Typescript error: TS2312: An interface can only extend an object

```
node_modules/smooth-scroll-into-view-if-needed/dist/index.d.ts:32:56 - error TS2312: An interface can only extend an object type or intersection of object types with statically known members.

32 export declare interface SmoothBehaviorOptions extends Options {
```


 This is because you cannot extend a union type using an interface. You must use type alias:
export type Options = { x: number } | { y: number }
export type SmoothBehaviorOptions = Options & { z: number }

--- a/src/index.ts
+++ b/src/index.ts
@@ -16,7 +16,7 @@ type OnScrollChangeCallback = (scrollState: {
   top: number
 }) => void
 
-export interface SmoothBehaviorOptions extends Options {
+export type SmoothBehaviorOptions = Options & {
   behavior?: 'smooth'
   duration?: number
   ease?: CustomEasing

